### ISSUE TITLE
Warning: Label 'page load time' already exists for console.time()

### ISSUE THREAD

**Issue by gpaciga** [dequelabs/axe-core-npm#208]
When using timers, an warning is reported:

```
Warning: Label 'page load time' already exists for console.time()
```

It looks like this might just be because the timer is set twice in `packages/cli/src/lib/axe-test-urls.ts`?
* https://github.com/dequelabs/axe-core-npm/blob/develop/packages/cli/src/lib/axe-test-urls.ts#L26
* https://github.com/dequelabs/axe-core-npm/blob/develop/packages/cli/src/lib/axe-test-urls.ts#L49

```
    if (config.timer) {
      events?.startTimer('page load time');
    }
```

Are these meant to be two different timers or is this just an error?

In either case, it is probably wise to make the name more specific, (e.g. `axe page load time`) to avoid the potential for conflicts with non-axe timers (which is what I thought this was originally from).

**Comment by padmavemulapati** [comment]
seeing the difference in the `axe-test-ulrs.ts` file - instead of ` page load time` , updated as `axe page load time` so that it is looking for the time to load only for the test page after triggering axe

**Comment by michael-siek** [comment]
Closing no docs required.
